src/main/java/Butler.java

```diff
@@ -0,0 +1,216 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class Butler {
+
+    private static void addTask(String input, ArrayList<Task> taskList, Boolean isNew) {
+        String[] inputList = input.split(" ", 2);
+        String reply = "";
+        Task addedTask;
+        String summary;
+        Boolean isValidCommand = false;
+        try {
+            switch (inputList[0]) {
+                case "todo":
+                    try {
+                        addedTask = new ToDoTask(inputList[1]);
+                        taskList.add(addedTask);
+                        reply += "\nI have added your ToDo task as follows:\n"
+                                + "Added: " + addedTask + "\n";
+                        isValidCommand = true;
+                    } catch (Exception e) {
+                        System.out.println("\nExcuse me, " +
+                                "but I will need a description for the ToDo task given.\n");
+                    }
+                    break;
+
```

Comment:
Should there be indentations for case clauses?

Reply:
Thank you for pointing this out. I have removed the indentations to adhere to coding standard.